ios/kmbuild.sh

```diff
@@ -169,28 +170,27 @@ update_bundle ( ) {
           KMWFLAGS="$KMWFLAGS -upload-sentry"
         fi
 
-        ./build.sh $KMWFLAGS
+        $KEYMAN_ROOT/web/build.sh $KMWFLAGS
```

Comment:
```suggestion
        "$KEYMAN_ROOT/web/build.sh" $KMWFLAGS
```

We should be fixing these unquoted paths when we find them -- there's quite a few in the iOS and Android build.sh files

Reply:
... sure.  There have been a few times on prior PRs where you un-quoted things, though, so I'm a bit ambiguous on exactly when it matters and for which cases.

Reply:
It matters for anything which could have spaces, such as `$KEYMAN_ROOT`, which could be `/c/Users/my name/Documents/keyman/` for instance. I will tend to remove quotes around single-word strings to reduce unnecessary clutter.